.github/workflows/agentic-tests.yml

```
@@ -0,0 +1,246 @@
+        id: skill_test
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+          prompt: |
+            First, read the skill definition file to understand your role:
+            .claude/skills/${{ matrix.skill }}/SKILL.md
+
+            After reading and adopting that persona, respond to this prompt:
+            ${{ matrix.prompt }}
+
+            Keep your response focused and concise.
+          allowed_tools: "Read"
```

actionlint (1.7.9)

input "allowed_tools" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_prefix", "claude_args", "claude_code_oauth_token", "experimental_allowed_domains", "github_token", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_sticky_comment", "use_vertex"

(action)